authentication/src/test/java/io/camunda/authentication/CamundaOidcUserServiceTest.java

```diff
@@ -49,13 +52,25 @@ public class CamundaOidcUserServiceTest {
   @Mock private RoleServices roleServices;
   @Mock private GroupServices groupServices;
   @Mock private AuthorizationServices authorizationServices;
+  @Mock private SecurityConfiguration securityConfiguration;
+  @Mock private AuthenticationConfiguration authenticationConfiguration;
+  @Mock private OidcAuthenticationConfiguration oidcAuthenticationConfiguration;
```

Comment:
:upside_down_face: These are all simple data holders, setting the right values seems easier than setting up mocks

Reply:
Yea I also tried that approach too but then its a case of creating the objects and setting them to longer term it doesn't really add any benefit, this way the code is the same through the class (using mocks)